packages/playground/src/components/account/account-register/account-register.tsx

```diff
@@ -1,10 +1,32 @@
+declare var web3: {
```

Comment:
I think this is a good candidate to be placed inside `typescript-typings`

Reply:
https://www.npmjs.com/package/@types/web3 also exists

Reply:
Unless this is so insignificant so as to not warrant a dependency you think?

Reply:
@snario Yeah, I agree. The `@types/web3` package isn't compatible with the Metamask web3 version. It's a shame :man_shrugging: